src/runtime/providers/gumlet.ts

```diff
@@ -0,0 +1,90 @@
+import { joinURL } from 'ufo'
+import type { ProviderGetImage } from 'src'
+import { createOperationsGenerator } from '~image'
+
+export const operationsGenerator = createOperationsGenerator({
+  keyMap: {
+    width: 'w',
+    height: 'h',
+    format: 'format',
+    quality: 'q',
+    backgroundColor: 'bg',
+    rotate: 'rot',
+    mask: 'mask',
+    auto: 'auto',
+    crop: 'crop',
+    brightness: 'bri',
+    contrast: 'con',
+    exposure: 'exp',
+    gamma: 'gam',
+    highlight: 'high',
+    hueShift: 'hue',
+    invert: 'invert',
+    saturation: 'sat',
+    sharpen: 'sharp',
+    padding: 'pad',
+    paletteColorCount: 'colors',
+    colorPaletteExtraction: 'palette',
+    cssPrefix: 'prefix',
+    jsonFaceData: 'faces',
+    fillMode: 'fill',
+    fillColor: 'fill-color',
+    transparency: 'transparency',
+    focalPointDebug: 'fp-debug',
+    focalPointXPosition: 'fp-x',
+    focalPointYPosition: 'fp-y',
+    focalPointZoom: 'fp-z',
+    chromaSubsampling: 'chromasub',
+    colorQuantization: 'colorquant',
+    colorSpace: 'cs',
+    download: 'dl',
+    dotsPerInch: 'dpi',
+    losslessCompression: 'lossless',
+    maskBackgroundColor: 'mask-bg',
+    maskCornerRadius: 'corner-radius',
+    pdfPageNumber: 'page',
+    pixelDensity: 'dpr',
+    orientation: 'orient',
+    flipAxis: 'flip',
+    aspectRatio: 'ar',
+    maximumHeight: 'max-h',
+    maximumWidth: 'max-w',
+    minimumHeight: 'min-h',
+    minimumWidth: 'min-w',
+    sourceRectangleRegion: 'rect',
+    monochrome: 'monochrome',
+    pixellate: 'px',
```

Comment:
@adityapatadia May I ask how did you come up with all these image transformations?
I'm asking only because i cannot find them inside the gumlet documentation.
For example I cannot find `pixellate` anywhere inside https://docs.gumlet.com/reference/image-operations.

Reply:
We have actually imgix compatible APIs and some of parameters are not fully documented. I have removed those parameters from this PR and kept only documented ones.

Please let me know if you need further help.

Reply:
Hope it's good to merge after last change.

Reply:
I'm not really responsible to judge your pr.
I'm just using gumlet in my project and i was curious how to use all these transformations.
I really would love to use pixellate for example

Reply:
Oh okay, can you please send your required parameters to support@gumlet.com? We will get them ready asap for you.